dockerised builds fail with unknown instruction: -E

Apparently a '-e' in the echo command in the Makefile for all `docker build` commands causes an error

I cannot reproduce this.
In my case, the opposite happens: docker build fails _without_ -e (which enables interpretation of backslash escapes):

```
$ make dockerised_deb_debian_buster
echo "\ndebian_buster"
\ndebian_buster
Sending build context to Docker daemon  232.2MB
Error response from daemon: dockerfile parse error line 1: FROM requires either one or three arguments
make: *** [Makefile:57: docker_debian_buster] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -10,6 +10,7 @@ DOCKER_BASE=`dirname ${PWD}`
 PACKAGE=`basename ${PWD}`
 SRC_TAR:=$(PKG_NAME).tar
 
+SHELL:=bash
 
 ### Actual targets
 default: sdist bdist_wheel
@@ -54,16 +55,16 @@ dockerised_all_packages: dockerised_deb_debian_buster dockerised_deb_debian_bull
 docker_images: docker_centos8 docker_debian_bullseye docker_debian_buster docker_ubuntu_bionic docker_ubuntu_focal
 docker_debian_buster:
 	echo "\ndebian_buster"
-	@echo "FROM debian:buster\n"\
+	echo -e "FROM debian:buster\n"\
 	"RUN apt-get update && "\
 		"apt-get -y upgrade && "\
 		"apt-get -y install build-essential dh-make quilt "\
 		"python3-virtualenv dh-virtualenv python3-venv devscripts git "\
-    	"python3 python3-dev python3-pip python3-setuptools "| \
+    	"python3 python3-dev python3-pip python3-setuptools " | \
 	docker build --tag debian_buster -f - .
 docker_debian_bullseye:
 	echo "\ndebian_bullseye"
-	@echo "FROM debian:bullseye\n"\
+	@echo -e "FROM debian:bullseye\n"\
 	"RUN apt-get update && "\
 		"apt-get -y upgrade && "\
 		"apt-get -y install build-essential dh-make quilt "\
@@ -72,7 +73,7 @@ docker_debian_bullseye:
 	docker build --tag debian_bullseye -f - .
 docker_ubuntu_bionic:
 	echo "\nubuntu_bionic"
-	@echo "FROM ubuntu:bionic\n"\
+	@echo -e "FROM ubuntu:bionic\n"\
 	"RUN apt-get update && "\
 		"apt-get -y upgrade && "\
 		"apt-get -y install build-essential dh-make quilt "\
@@ -81,7 +82,7 @@ docker_ubuntu_bionic:
 	docker build --tag ubuntu_bionic -f - .
 docker_ubuntu_focal:
 	echo "\nubuntu_focal"
-	@echo "FROM ubuntu:focal\n"\
+	@echo -e "FROM ubuntu:focal\n"\
 	"ENV DEBIAN_FRONTEND=noninteractive\n"\
 	"ENV  TZ=Europe/Berlin\n"\
 	"RUN apt-get update && "\
@@ -92,7 +93,7 @@ docker_ubuntu_focal:
 	docker build --tag ubuntu_focal -f - .
 docker_centos8:
 	echo "\ncentos8"
-	@echo "FROM centos:8\n"\
+	@echo -e "FROM centos:8\n"\
 	"RUN yum install -y make rpm-build\n" \
 	"RUN dnf -y group install \"Development Tools\"\n" | \
 	docker build --tag centos8 -f -  .

--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,7 @@ dockerised_all_packages: dockerised_deb_debian_buster dockerised_deb_debian_bull
 
 docker_images: docker_centos8 docker_debian_bullseye docker_debian_buster docker_ubuntu_bionic docker_ubuntu_focal
 docker_debian_buster:
-	echo "\ndebian_buster"
+	echo -e "\ndebian_buster"
 	echo -e "FROM debian:buster\n"\
 	"RUN apt-get update && "\
 		"apt-get -y upgrade && "\
@@ -63,7 +63,7 @@ docker_debian_buster:
     	"python3 python3-dev python3-pip python3-setuptools " | \
 	docker build --tag debian_buster -f - .
 docker_debian_bullseye:
-	echo "\ndebian_bullseye"
+	echo -e "\ndebian_bullseye"
 	@echo -e "FROM debian:bullseye\n"\
 	"RUN apt-get update && "\
 		"apt-get -y upgrade && "\
@@ -72,7 +72,7 @@ docker_debian_bullseye:
 		"python3 python3-dev python3-pip python3-setuptools "| \
 	docker build --tag debian_bullseye -f - .
 docker_ubuntu_bionic:
-	echo "\nubuntu_bionic"
+	echo -e "\nubuntu_bionic"
 	@echo -e "FROM ubuntu:bionic\n"\
 	"RUN apt-get update && "\
 		"apt-get -y upgrade && "\
@@ -81,7 +81,7 @@ docker_ubuntu_bionic:
 		"python3 python3-dev python3-pip python3-setuptools "| \
 	docker build --tag ubuntu_bionic -f - .
 docker_ubuntu_focal:
-	echo "\nubuntu_focal"
+	echo -e "\nubuntu_focal"
 	@echo -e "FROM ubuntu:focal\n"\
 	"ENV DEBIAN_FRONTEND=noninteractive\n"\
 	"ENV  TZ=Europe/Berlin\n"\
@@ -92,7 +92,7 @@ docker_ubuntu_focal:
 		"python3 python3-dev python3-pip python3-setuptools "| \
 	docker build --tag ubuntu_focal -f - .
 docker_centos8:
-	echo "\ncentos8"
+	echo -e "\ncentos8"
 	@echo -e "FROM centos:8\n"\
 	"RUN yum install -y make rpm-build\n" \
 	"RUN dnf -y group install \"Development Tools\"\n" | \

--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,7 @@ dockerised_all_packages: dockerised_deb_debian_buster dockerised_deb_debian_bull
 docker_images: docker_centos8 docker_debian_bullseye docker_debian_buster docker_ubuntu_bionic docker_ubuntu_focal
 docker_debian_buster:
 	echo "\ndebian_buster"
-	@echo -e "FROM debian:buster\n"\
+	@echo "FROM debian:buster\n"\
 	"RUN apt-get update && "\
 		"apt-get -y upgrade && "\
 		"apt-get -y install build-essential dh-make quilt "\
@@ -63,7 +63,7 @@ docker_debian_buster:
 	docker build --tag debian_buster -f - .
 docker_debian_bullseye:
 	echo "\ndebian_bullseye"
-	@echo -e "FROM debian:bullseye\n"\
+	@echo "FROM debian:bullseye\n"\
 	"RUN apt-get update && "\
 		"apt-get -y upgrade && "\
 		"apt-get -y install build-essential dh-make quilt "\
@@ -72,7 +72,7 @@ docker_debian_bullseye:
 	docker build --tag debian_bullseye -f - .
 docker_ubuntu_bionic:
 	echo "\nubuntu_bionic"
-	@echo -e "FROM ubuntu:bionic\n"\
+	@echo "FROM ubuntu:bionic\n"\
 	"RUN apt-get update && "\
 		"apt-get -y upgrade && "\
 		"apt-get -y install build-essential dh-make quilt "\
@@ -81,7 +81,7 @@ docker_ubuntu_bionic:
 	docker build --tag ubuntu_bionic -f - .
 docker_ubuntu_focal:
 	echo "\nubuntu_focal"
-	@echo -e "FROM ubuntu:focal\n"\
+	@echo "FROM ubuntu:focal\n"\
 	"ENV DEBIAN_FRONTEND=noninteractive\n"\
 	"ENV  TZ=Europe/Berlin\n"\
 	"RUN apt-get update && "\
@@ -92,7 +92,7 @@ docker_ubuntu_focal:
 	docker build --tag ubuntu_focal -f - .
 docker_centos8:
 	echo "\ncentos8"
-	@echo -e "FROM centos:8\n"\
+	@echo "FROM centos:8\n"\
 	"RUN yum install -y make rpm-build\n" \
 	"RUN dnf -y group install \"Development Tools\"\n" | \
 	docker build --tag centos8 -f -  .

